adaptors/hybrisproximityadaptor/hybrisproximityadaptor.cpp

```diff
@@ -96,7 +95,7 @@ void HybrisProximityAdaptor::sendInitialData()
                ProximityData *d = buffer->nextSlot();
                d->timestamp_ = Utils::getTimeStamp();
                d->withinProximity_ = near;
-               d->value_ = near ? 10 : 0;
+               d->value_ = absinfo.value;
```

Comment:
The fd is not needed again. The changed value comes from hybris. The issue is that android has no way to get current sensor value, and does not send value initially.

Point taken regarding failure and closed proximity reading.